test/SlowTests/Issues/RavenDB_12151.cs

```diff
@@ -116,7 +116,7 @@ public void IndexingWhenGlobalScratchSpaceLimitExceeded32()
             }
         }
 
-        [Fact]
+        [Fact(Skip ="Transaction size changed below 1MB")]
```

Comment:
Why this change?

Reply:
What changed the TX size? We need to fix the test / figure out why it changed

Reply:
hard to track what caused it, @maximburyak and I decided to finish fixing the rest of the tests and try to figure this one on a later PR since it was taking too long and I couldn't track what caused it.
I know for sure this test was passing when I wrote it.
now the largest Tx is 334kb...

Reply:
It is likely the limitations I have put on the merger that are limiting the indexing transactions. I'll open a separate issue to investigate this but for the short term let's get the tests green again.

Reply:
It is likely the limitations I have put on the merger that are limiting the indexing transactions. I'll open a separate issue to investigate this but for the short term let's get the tests green again.

Reply:
That was resolved in the commit below